CODIGO/ModAuth.bas

```diff
@@ -543,6 +550,64 @@ Public Sub HandleLogOutRequest(ByVal bytesTotal As Long)
         End Select
     End If
     Auth_state = e_state.Idle
+End Sub
+Public Sub SendRequestTransferCharacter()
+    Dim json As String
+    Dim len_encrypted_username As Integer
+    Dim login_request() As Byte
```

Comment:
should we rename this to transfer_request?

Reply:
sure